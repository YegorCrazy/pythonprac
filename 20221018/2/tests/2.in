:one 1
one
quit "func: {}, lines: {}"
