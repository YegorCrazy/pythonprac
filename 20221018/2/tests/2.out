1
func: 2, lines: 3
